eng/pipelines/coreclr/templates/perf-job.yml

```diff
@@ -47,21 +47,25 @@ jobs:
     - ${{ format('coreclr_{0}_product_build_{1}{2}_{3}_{4}', parameters.runtimeVariant, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
     - ${{ if ne(parameters.liveLibrariesBuildConfig, '') }}:
       - ${{ format('libraries_build_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.liveLibrariesBuildConfig) }}
-    - ${{ if eq(parameters.runtimeType, 'mono') }}:
+    - ${{ if and(eq(parameters.runtimeType, 'mono'), ne(parameters.codeGenType, 'AOT')) }}:
       - ${{ format('mono_{0}_product_build_{1}{2}_{3}_{4}', parameters.runtimeVariant, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
     - ${{ if eq(parameters.runtimeType, 'wasm') }}:
       - ${{ format('build_{0}{1}_{2}_{3}_{4}', 'Browser', '', 'wasm', parameters.buildConfig, parameters.runtimeType) }}
+    - ${{ if eq(parameters.codeGenType, 'AOT')}}:
```

Comment:
nit: Will there ever be AOT runs that are not Mono? (Might we use this to indicate some sort of crossgenning for scenarios?) Maybe this should check for both `mono` and `AOT`?

Reply:
I thought about this as well. I am going to make an issue to refactor this, because it is a bit of a mess. As it stands right now if the mono parameter gets passed it will do work that we do not want it to do. But fixing that should not block getting AOT in.